### DISCUSSION TITLE
Installing for Arduino IDE 2.x?

### DISCUSSION THREAD

**Discussion by JetForMe** [pschatzmann/arduino-audio-tools#1777] [Q&A] [answered]
How do I install this in the 2.x Arduino IDE? I tried cloning it into Arduino/libraries, but I think that's only for Arduino 1.x. At least, it's not showing up for me.

Interestingly, I can actually include the headers, but I don't see the examples.

**Comment by pschatzmann** [comment, marked as answer]
There is no change!

Maybe you just copied it to the wrong location: You can find the Sketchbook location in the Preferences.
After you install a library you need to restart Arduino to load the new examples...

**Reply by JetForMe** [reply]
I did try restarting the IDE, but I still have no arduino audio tool examples.
